src/Controllers/Stats/fullTracklist.php

```diff
@@ -46,9 +46,9 @@
                 'Stats',
                 'fullTracklist',
                 [
-                    'rangesel-starttime' => array_key_exists($_GET['rangesel-starttime']) ?
+                    'rangesel-starttime' => isset($_GET['rangesel-starttime']) ?
                         $_GET['rangesel-starttime'] : '',
-                    'rangesel-endtime' => array_key_exists($_GET['rangesel-endtime']) ? $_GET['rangesel-endtime'] : '',
+                    'rangesel-endtime' => isset($_GET['rangesel-endtime']) ? $_GET['rangesel-endtime'] : '',
```

Comment:
Hmm, I guess, but I wanted to make it obvious. It's only used in the fullTrackList / csv from what I can see.